### ISSUE TITLE
Mint 20: When purging or downgrading foreign packages, I'm dropped at a shell prompt after the command finishes

### ISSUE THREAD

**Issue by hamishmb** [linuxmint/mintsources#213]
After purging/downgrading foreign packages I'm dropped at a shell prompt instead of the dialog closing. Not clear if this is intended, but it doesn't seem like great user experience.

**Comment by xenopeek** [comment]
This is reproducible with mintsources 2.0.5. The prompt just sits there, without a blinking cursor, waiting for the user to close the dialog. It would be nicer if there is a notice like "You may now close the dialog" or something after the command finishes.

![image](https://user-images.githubusercontent.com/21159972/101162861-29f3af00-3633-11eb-94e0-ae314a0fb044.png)

**Comment by hamishmb** [comment]
Yeah, I mean you're already doing a somewhat scary thing as instructed by the upgrade guide - it'd be even worse as a new user if something like this happened.

**Comment by clefebvre** [comment]
This was fixed quite a while ago.

**Comment by hamishmb** [comment]
Thanks for letting me know :)
